packages/server/README.md

```diff
@@ -0,0 +1 @@
+https://www.postgresql.org/docs/current/mvcc-serialization-failure-handling.html
```

Comment:
A little too concise :D 

Why linking this page?

Reply:
Removed this, was just a reminder for myself.